api/v1alpha/client_example.go

```diff
@@ -51,6 +51,24 @@ func main() {
 
 	for _, p := range podResp.Pods {
 		fmt.Printf("Pod %q is running\n", p.Id)
+
+		logsResp, err := c.GetLogs(context.Background(), &v1alpha.GetLogsRequest{
+			PodId: p.Id,
+		})
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(2)
```

Comment:
yeah, sequential numbering never works, settle on 1 unless you wanted to
try use something like http://linux.die.net/include/sysexits.h

On Fri, May 27, 2016 at 2:21 PM, Iago López Galeiras <
notifications@github.com> wrote:

> In api/v1alpha/client_example.go
> <https://github.com/coreos/rkt/pull/2662#discussion_r64896509>:
>
> > @@ -51,6 +51,24 @@ func main() {
> >
> >  	for _, p := range podResp.Pods {
> >  		fmt.Printf("Pod %q is running\n", p.Id)
> > +
> > +		logsResp, err := c.GetLogs(context.Background(), &v1alpha.GetLogsRequest{
> > +			PodId: p.Id,
> > +		})
> > +		if err != nil {
> > +			fmt.Println(err)
> > +			os.Exit(2)
>
> Dunno, this stupid error numbering. I guess it should be 3, and then 4,
> and then 5. I can just get rid of those like we did with init.go some time
> ago
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreos/rkt/pull/2662/files/c28aa39ba2c92941f58487762ed66bf9e93d0d01..cc72b0fb026b3fb54a9a9bb24e29f5ed86f8df50#r64896509>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACewN0s-tIsx_Ge3uE7wnpSOHppyL1lMks5qFuHKgaJpZM4IiF05>
> .
>